packages/interledger/src/admin-api/schema.graphql

```diff
@@ -1,22 +1,437 @@
 type Query {
-  accounts: String
+  "Fetch a page of Interledger accounts."
+  ilpAccounts(
+    "Paginating forwards: the cursor before the the requested page."
+    after: String
+    "Paginating backwards: the cursor after the the requested page."
+    before: String
+    "Paginating forwards: The first **n** elements from the page."
+    first: Int
+    "Paginating backwards: The last **n** elements from the page."
+    last: Int
+  ): IlpAccountsConnection!
+
+  "Get an Interledger account by ID."
+  ilpAccount(id: ID!): IlpAccount!
+
+  "Get a trustline by ID."
+  trustline("The ID of the trustline to get." id: ID!): Trustline!
+
+  "Get a webhook by ID."
+  webhook("The ID of the webhook to get." id: ID!): Webhook!
+
+  "Get a deposit by ID."
+  deposit("The ID of the deposit to get." id: ID!): Deposit!
+
+  "Get a withdrawal by ID."
+  withdrawal("The ID of the withdrawal to get." id: ID!): Withdrawal!
+}
+
+type Mutation {
+  "Create Interledger account"
+  createIlpAccount: CreateIlpAccountMutationResponse!
+
+  "Update Interledger account"
+  updateIlpAccount: UpdateIlpAccountMutationResponse!
+
+  "Delete Interledger account"
+  deleteIlpAccount: DeleteIlpAccountMutationResponse!
+
+  "Create Interledger sub-account"
+  createIlpSubAccount(superAccountId: ID!): CreateIlpSubAccountMutationResponse!
+
+  "Transfer between Interledger accounts"
+  transfer(
+    originAmount: Int!
+    originAccountId: ID!
+    destinationAccountId: ID!
+    destinationAmount: Int
+    autoCommit: Boolean
+    idempotencyKey: ID!
+  ): TransferMutationResponse
+
+  "Provision trustline"
+  provisionTrustline: ProvisionTrustlineMutationResponse
```

Comment:
What does this do?

Reply:
I'm not sure what you're referring to?

Reply:
What does it mean to 'provision' a trustline? How is it different than creating a sub-account? Why does this call have no parameters?

Reply:
> TODO — Provision trustline IDs for operator and/or super-account at the time of ILP account creation, but don't create TigerBeetle accounts until `extend` operation is called?

Reading it more carefully I see we don't need this at all. I left out the parameters because it was TODO in the docs.